### DISCUSSION TITLE
General Questions by Stephen

### DISCUSSION THREAD

**Discussion by StephenHillenbrand** [MarkW06071969/Tafe202206#22] [General]
Hi all i have some general questions that we may have already discussed but I will ask them again here if someone could please respond

1. Who is going to merge the release branch into the master branch?
2. Are all the buttons on the application going to use the same colour?
3. By what date should the release branch be merged into the master branch?

Thanks in advance
Stephen

**Comment by MarkW06071969** [comment]
1. Hi Stephen, if you could do it, that would be great!
2. yes, the overall look look of the application should have some similarity.
3.  by September 8th.
